src/event-db/src/types/event/proposal.rs

```diff
@@ -45,6 +45,19 @@ pub struct Proposal {
     pub proposal_details: ProposalDetails,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct EventId(pub i32);
```

Comment:
Duplication of this type https://github.com/input-output-hk/catalyst-core/blob/0444b08ec66c80ae95cf8bbdbe606d0ed1bd6a06/src/event-db/src/types/event/mod.rs#L10